library/roc_consumer/src/request/request.e

```diff
@@ -0,0 +1,207 @@
+note
+	description: "Summary description for {REQUEST}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
```

Comment:
Maybe you could suggest improvements in http_client.
And having those code in a clear library on top of http client, would help
reviewing it alone, and not among other changes.



2015-10-02 15:24 GMT+02:00 Javier Velilla <notifications@github.com>:

> In library/roc_consumer/src/request/request.e
> <https://github.com/EiffelWebFramework/ROC/pull/43#discussion_r41020240>:
>
> > @@ -0,0 +1,207 @@
> > +note
> > +	description: "Summary description for {REQUEST}."
> > +	date: "$Date$"
> > +	revision: "$Revision$"
> > +
> > +class
>
> I think I can use http_client directly, but I wanted something simpler to
> use.
> About persistent connection, yes is not taking care of it and I agree, I
> will create an extension on top of HTTP client to avoid duplication.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EiffelWebFramework/ROC/pull/43/files#r41020240>.
>